client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -139,18 +140,47 @@ class BusinessDetails extends Component {
 		return keys( pickBy( values ) ).filter( name => this.extensions.includes( name ) );
 	}
 
-	getNumberRangeString( min, max = false ) {
+	convertCurrency( value ) {
+		// These are rough exchange rates from USD.  Precision is not paramount.
+		// The keys here should match the keys in `getCurrencyData`.
+		const exchangeRates = {
+			US: 1,
+			EU: 0.9,
+			IN: 71.24,
+			GB: 0.76,
+			BR: 4.19,
+			VN: 23172.5,
+			ID: 14031.0,
+			BD: 84.87,
+			PK: 154.8,
+			RU: 63.74,
+			TR: 5.75,
+			MX: 19.37,
+			CA: 1.32,
+		};
+
+		const region = getCurrencyRegion( this.props.settings.woocommerce_default_country );
+		const exchangeRate = exchangeRates[ region ] || exchangeRates.US;
+
+		return Math.round( value * exchangeRate / 1000 ) * 1000;
```

Comment:
Nice!  Optional, but we could add a zero for every additional digit beyond the first digit to round currencies with larger differences in exchange rates.

E.g., MX would be `10000`, VND would become `10000000`.

This might make the numbers a bit more rounded some currencies:

**Before**
<img width="675" alt="Screen Shot 2019-12-09 at 1 41 51 PM" src="https://user-images.githubusercontent.com/10561050/70413515-2bbf7c80-1a8a-11ea-9258-97b85120d1f3.png">

**After**
<img width="474" alt="Screen Shot 2019-12-09 at 1 46 43 PM" src="https://user-images.githubusercontent.com/10561050/70413611-63c6bf80-1a8a-11ea-85f8-fba5427143c3.png">

Reply:
Nice suggestion @joshuatf! Handled in https://github.com/woocommerce/woocommerce-admin/pull/3358/commits/df799125fd068ed477d9a4ad789bbe2345444783.